### PR TITLE
Add Million Token (MM)

### PR DESCRIPTION
**Pull request by imsys**
## I agree to not ping *anybody* on Discord/Twitter/email about this pull request. Instead I will inquire by posting a new comment in the pull request if needed.
----
PRs are reviewed in bulk and and can take up to **two weeks** to be merged.  

**Please provide the following information for your token.**

Please include change to the `src/tokens/solana.tokenlist.json` file in the PR. 
DON'T modify any other token on the list.

At minimum each entry should have
* Token Address: FiCiuX9DetEE89PgRAU1hmoptnem8b1fkpEq8PGYTYkd
* Token Name: Million Token
* Token Symbol: MM
* Logo: In the commit
* Link to the official homepage of token: https://www.milliontoken.org/
* Coingecko ID if available (https://www.coingecko.com/api/documentations/v3#/coins/get_coins__id_): million

### CODE REVIEW

File: src/tokens/solana.tokenlist.json [hunk 1]
```diff
@@ -11915,7 +11915,32 @@
         "telegram": "https://t.me/kurobi_io",
         "twitter": "https://twitter.com/kurobi_io"
       }
-    } 
+    },
+    {
+      "chainId": 101,
+      "address": "FiCiuX9DetEE89PgRAU1hmoptnem8b1fkpEq8PGYTYkd",
+      "symbol": "MM",
+      "name": "Million Token",
```
Comment (techleadhd) [inline src/tokens/solana.tokenlist.json:11923]: Can you please change name to "Million"? Thanks!

Comment (imsys) [reply src/tokens/solana.tokenlist.json]: Done!

**Comment by imsys** [comment]
I did a mistake while merging, I will fix that.

**Comment by imsys** [comment]
#881

**Pull request closed without merging**